src/content/documentation/develop/firefox-workflow-overview.md

```diff
@@ -66,7 +66,7 @@ date: 2020-03-24 12:47:00
         </tr>
         <tr>
             <td><a href="https://webext.eu/">Create your extension scaffold</a></td>
-            <td>Debug with the <a href="https://developer.mozilla.org/docs/Tools/Browser_Toolbox/">Add-on Debugging Window</a></td>
+            <td>Debug with the <a href="https://firefox-source-docs.mozilla.org/devtools-user/browser_toolbox/index.html">Add-on Debugging Window</a></td>
```

Comment:
I haven't seen the Browser Toolbox referred to as the "Add-on Debugging Window" before. If we have prior art here, please correct me.

I suggest we use the more common term for linked feature or update the link (and name) to reference the [add-on specific developer tools window](https://extensionworkshop.com/documentation/develop/debugging/#developer-tools-toolbox) exposed in about:debugging.

```suggestion
            <td>Debug with Firefox's <a href="https://firefox-source-docs.mozilla.org/devtools-user/browser_toolbox/index.html">Browser Toolbox</a></td>
```